lib/stat_tracker.rb

```diff
@@ -32,4 +32,68 @@ def highest_total_score
   def lowest_total_score
     @game_csv.map { |row| row[:away_goals].to_i + row[:home_goals].to_i }.min
   end
-end
\ No newline at end of file
+
+  def percentage_home_wins
```

Comment:
Is this for all home teams or should this have an argument to lookup an individual home team's stats?

Edit: looks like the spec_harness only tests for all home teams, but passing an argument might be useful depending on later methods